README.md

```diff
@@ -138,6 +144,38 @@ Open Crawler has a Dockerfile that can be built and run locally.
 
 See [CONFIG.md](docs/CONFIG.md) for in-depth details on Open Crawler configuration files.
 
+### Scheduling Recurring Crawl Jobs
+
+Crawl jobs can also be scheduled to recur.
+Scheduled crawl jobs run until terminated by the user.
+
+These schedules are defined through a cron expression.
+This expression needs to be included in the Crawler config file.
+You can use the tool https://crontab.guru to test different cron expressions.
+Crawler supports all standard cron expressions.
+
+See an example below for a crawl schedule that will execute once every 30 minutes.
+
+```yaml
+domains:
+  - url: "https://elastic.co"
+schedule:
+  - interval: "*/30 * * * *" # run every 30th minute
+```
+
+Then, use the CLI to then begin the crawl job schedule:
+
+```bash
+docker exec -it crawler bin/crawler schedule path/to/my-crawler.yml
+```
+
+**Scheduled crawl jobs from a single execution will not overlap.**
+If you have a schedule that triggers once per minute, but your crawl job takes 5 minutes to complete, the crawl schedule will effectively trigger about every 6 minutes.
```

Comment:
What if the crawl takes 30s and the schedule is every minute? Will it get scheduled every minute? Or every 1.5 minutes?

What if the crawl takes 5 hrs and we schedule it every Friday at 12? Will the next crawl be shifted by 5hrs, e.g. start at 5pm? 

> If you have a schedule that triggers once per minute, but your crawl job takes 5 minutes to complete, the crawl schedule will effectively trigger about every 6 minutes.

I'm just clarifying because CRON schedules stuff based on specific times (e.g., “every Friday at 12 PM”), not intervals (e.g., “every 5 minutes after the last job completes”).  If a job is still running when the next scheduled time arrives, CRON doesn’t automatically shift the schedule; it simply waits until the job completes before starting the next one.

Reply:
> What if the crawl takes 30s and the schedule is every minute? Will it get scheduled every minute? Or every 1.5 minutes?

It will be every minute. I think my example isn't clear, whether the second sync runs 5 minutes later or 6 minutes later will depend on if the first sync finished in the seconds before or after the 5 minute mark

> What if the crawl takes 5 hrs and we schedule it every Friday at 12? Will the next crawl be shifted by 5hrs, e.g. start at 5pm?

The next crawl shouldn't be shifted by 5 hours if they used an exact time expression instead of an interval expression (e.g. `0 12 * * 5`).

>  If a job is still running when the next scheduled time arrives, CRON doesn’t automatically shift the schedule; it simply waits until the job completes before starting the next one.

In the case of `rufus-scheduler`, it won't wait for the job to finish, because of the argument `overlap: false`. It will just not execute the next job if the current job is still executing.

Reply:
I updated the time example